kDrive/UI/View/Files/Preview/AudioCollectionViewCell.swift

```diff
@@ -86,6 +92,17 @@ class AudioCollectionViewCell: PreviewCollectionViewCell {
         setUpObservers()
     }
 
+    func setUpPlayButtons() {
+        if UIDevice.current.orientation.isPortrait {
```

Comment:
I would suggest using `UIApplication.shared.statusBarOrientation.isPortrait` to prevent having the edge case with `isFlat`